.drone/drone.yml

```diff
@@ -1537,6 +1557,6 @@ kind: secret
 name: gpg_private_key
 ---
 kind: signature
-hmac: 8738700b68f651859a25c2c9bd9f70efb8092aa1eca2ad35238a85ec19f9fe31
+hmac: 54378a177012c6a956ee0436d83365e6dd3a51fb424a2c331c3189efc2bf404e
```

Comment:
not sure what the correct value is here, but careful as I just broke `main` by removing it :cold_sweat: